packages/db/src/schema/workspace.ts

```diff
@@ -1,15 +1,24 @@
 import { relations, sql } from "drizzle-orm";
 import { integer, sqliteTable, text } from "drizzle-orm/sqlite-core";
+import { createSelectSchema } from "drizzle-zod";
+import { z } from "zod";
 
 import { page } from "./page";
 import { usersToWorkspaces } from "./user";
 
+const plan = ["free", "pro"] as const;
```

Comment:
On polish, we should use one source of truth. Exporting it inside of `@openstatus/plans` could be a thing.